NEXT_CHANGELOG.md

```diff
@@ -56,6 +56,18 @@ This is a breaking change since slightly invalid input might have validated befo
 
 By [@o0Ignition0o](https://github.com/o0Ignition0o) in https://github.com/apollographql/router/pull/1211
 
+### Entry point improvements ([PR #1227](https://github.com/apollographql/router/pull/1227))
+`ApolloRouterBuilder` has been migrated to `buildstructor` for consistency with other code.
+Calls to `ApolloRouterBuilder::default()` should be migrated to `ApolloRouter::builder`.
+`FederatedServerHandle` has been renamed to `ApolloRouterHandle`.
+
+Removed functionality:
+* The ability to supply your own `RouterServiceFactory`. This may be added back if there is a concrete use case for it.
```

Comment:
I'd remove the second sentence.

Reply:
Fixed in [992ddd6](https://github.com/apollographql/router/pull/1227/commits/992ddd6ae411c6fdb4b9a5d39a21132ff8a00c48)